src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,41 @@
+package mate.controller;
```

Comment:
we could create a new packages in controller (for exmaple controller.driver.(create, delete...)